buildSrc/src/main/java/Core.kt

```diff
@@ -0,0 +1,4 @@
+object Core {
```

Comment:
This needs another name, Core is too abstract and is a name for more important things and generals to the project. Maybe "SplashScreen"?

Reply:
Yes but, Core name is for file naming, seeing what you do with the others Object dependencies, naming the file as the top level domain dependency. 

Maybe, AndroidXCore or something like that? This object in the future should contain more dependencies from androidx.core